people-ops/parting-ways.html.md

```diff
@@ -7,25 +7,32 @@ meta_description: >
 
 It's always sad when we have to part ways with an employee, whether because they have decided to
 pursue other opportunities or because they're not a good fit. When this happens, though, we want to
-make sure the offboarding experience is as smooth as possible for everyone involved. 
+make sure the offboarding experience is as smooth as possible for everyone involved.
 
 ## Resignation
 
 The employment contract provides the worker with the option of withdrawing unilaterally from the
-contract by submitting his/her resignation. The 
-[resignation process](https://www.cliclavoro.gov.it/Cittadini/Pagine/Adempimenti.aspx) is remote and
-can be sent in various ways:
+contract by submitting his/her resignation.
 
-* Autonomously, on the [cliclavoro.gov.it](http://cliclavoro.gov.it) website for all those who 
+### International employees
+
+The contract carries a Term and Termination clause. The term varies depending on the deal, it's
+usually 60 days, but please check your case.
```

Comment:
`check your contract.` I think works a little better here, just to be super clear. 😄

Reply:
thank you!